flamingo/plugins/sphinx_themes/sphinx_theme.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import shutil
 import os
+import docutils
```

Comment:
I don't know if sorting imports (e.g. alphabetically) is a thing in the flamingo codebase. Maybe do it at a best-effort basis?

Reply:
Took out the big hammer here and reworked the whole import section while I was on it :-)